server/src/callbacks.mjs

```diff
@@ -75,11 +76,20 @@ Empirica.onNewBatch(async function ({ batch }) {
     try {
       console.log("fetching");
       const fetched = await (await axios.get(url)).data;
+      try {
+        marked.parse(fetched);
+      } catch (error) {
+        console.log("Unable to parse markdown");
+      }
+      if ((fetched.match(new RegExp("\\S", "g")) || []).length < 75) {
+        console.log("Less than 75 non whitespace characters");
```

Comment:
Now that I reflect on this, its a bit of a confusing error message. Maybe something like "Detected under 75 characters in the topic markdown - please check that your file was loaded properly". You might also log the text after the error. 

Lets also make this console.warn(), and not throw the error. That way we won't error out other experiments...